packages/expo-payments-stripe/ios/EXPaymentsStripe/EXTPSStripeManager.m

```diff
@@ -822,7 +822,8 @@ -(void)updateContactMethodErrors: (NSString *)errorMessage {
 -(void)displayErrorAlert: (NSException *) exception {
   [[self getViewController] dismissViewControllerAnimated:YES completion:^{
     [self resolveApplePayCompletion:PKPaymentAuthorizationStatusFailure];
-    [self rejectPromiseWithCode:exception.name message:exception.reason];
+    NSString *modifiedMessage = [NSString stringWithFormat:@"{\"graphQLErrors\": [{\"message\": %@}]}", exception.reason];
```

Comment:
it might be useful to format the reason to a jsonified string so that ther e is only one code path to handle errors

Reply:
sounds good. I'll give it a shot.